### ISSUE TITLE
Helm chart: Publish it as an helm repository and/or OCI repository

### ISSUE THREAD

**Issue by samip5** [zulip/docker-zulip#407]
Currently there doesn't seem to be a way to simply add it as a helm repository or OCI repository. It should be possible to deploy without going the git route.

**Comment by mikkeschiren** [comment]
Yes, that would be the best way forward - for my perspective OCI would be the way to go.

**Comment by alexmv** [comment]
This is next after I get #609 finished and merged.

**Comment by samip5** [comment]
It needs to be set to public as by default it's private.

**Comment by alexmv** [comment]
I've not made a release yet; I'll update the docs to point to it once I've done so.
